src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -1572,5 +1578,14 @@ internal override bool Equals(TypeSymbol t2, TypeCompareKind comparison)
                 nativeInteger.Equals(this, comparison) :
                 base.Equals(t2, comparison);
         }
+
+#nullable enable
+        internal bool IsSimpleProgram
+        {
+            get
+            {
+                return this.declaration.Declarations.Any(d => d is { IsSimpleProgram: true });
```

Comment:
>d is { IsSimpleProgram: true } [](http://example.com/codeflow?start=62&length=30)

Consider using `d.IsSimpleProgram` to match the other use above. Or replace both with a property on `MergedTypeDeclaration`.